dashboard/client/src/pages/job/JobRow.tsx

```diff
@@ -56,6 +57,29 @@ export const JobRow = ({
     }
   })();
 
+  const logsLink = (() => {
+    let link: string | undefined = undefined;
```

Comment:
nit: assignment of undefined is redundant
```suggestion
    let link: string | undefined;
```